conda_smithy/templates/README.md.tmpl

```diff
@@ -73,7 +73,7 @@ Current build status
 {%- set appveyor_url_name = appveyor_badge_name.replace('.', '-') %}
 
 Linux: [![Circle CI](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}}.svg?style=svg)](https://circleci.com/gh/{{github.user_or_org}}/{{github.repo_name}})
-OSX: [![TravisCI](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}.svg?branch=master)](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}) 
+OSX: [![TravisCI](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}}.svg?branch=master)](https://travis-ci.org/{{github.user_or_org}}/{{github.repo_name}})
```

Comment:
This is the only real whitespace change of note that wasn't in `master`. The rest were already here.